test/smoke/src/main.ts

```diff
@@ -80,6 +76,7 @@ console.warn = function suppressWebdriverWarnings(message) {
 
 const appName = "latestRNApp";
 const userDataDir = path.join(testVSCodeExecutableFolder, "userTmpFolder");
+const artifactsPath = path.join(repoRoot, "logArtifacts");
```

Comment:
Are you sure this is a good name for log directory? Won't it be confused with Azure DevOps build artifacts?